src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -48,24 +98,52 @@ export default function EnvironmentEditor() {
                 <EnvironmentBar />
             </div>
             <div className={classes.container}>
-                <div className={classes.emptyButtonsContainer}>
-                    <div className={classes.buttonContainer}>
-                        <Button
-                            startIcon={<AddIcon />}
-                            className={classes.button}
-                        >
-                            New Scene from Scratch
-                        </Button>
-                    </div>
-                    <div className={classes.buttonContainer}>
-                        <Button
-                            startIcon={<AccountBalanceIcon />}
-                            className={classes.button}
+                {scenes !== undefined && scenes.length !== 0 ? (
+                    <DragDropContext onDragEnd={onDragEnd}>
+                        <Droppable
+                            droppableId="droppable"
+                            direction="horizontal"
                         >
-                            New Scene from Template
-                        </Button>
+                            {(provided) => (
```

Comment:
I haven't seen this syntax before, is it equivalent to a conditional like `{provided ? (...) : null}`?

Reply:
No it's an arrow function. I found it in their sample: https://codesandbox.io/s/mmrp44okvj

Reply:
Yea I guess I was just confused how React is going to render this -- like where is this provided variable coming from? Perhaps it's a dnd feature or something